t/discovery/nacos.t

```diff
@@ -0,0 +1,152 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+use t::APISIX 'no_plan';
+
+repeat_each(1);
+log_level('info');
+no_root_location();
+no_shuffle();
+
```

Comment:
We can add this block to check the error log:
```
add_block_preprocessor(sub {
    my ($block) = @_;

    if (!$block->no_error_log) {
        $block->set_value("no_error_log", "[error]\n[alert]");
    }
});
```

It would be helpful for debugging the 503.

Reply:
After add this,it throws error when run test file in my local pc:
pattern "[error]" should not match any line in error.log but matches line "2021/04/08 05:39:35 [error] 25739#25739: *3 [lua] init.lua:462: http_access_phase(): failed to set upstream: no valid upstream node: nil, client: 127.0.0.1, server: localhost, request: \"GET /hello HTTP/1.1\", host: \"localhost\"" (req 0)